src/sentry_opsgenie/plugin.py

```diff
@@ -91,15 +96,9 @@ def notify_users(self, group, event, fail_silently=False):
         recipients = self.get_option('recipients', group.project)
         alert_url = self.get_option('alert_url', group.project)
 
-        message = getattr(group, 'message_short', group.message).encode('utf-8')
-
-        payload = {
-           'apiKey': api_key,
-           'message': message,
-           'source': 'Sentry',
-           'details': self.get_group_data(group, event)
-        }
+        payload = self.build_payload(group, event)
 
+        payload['api_key'] = api_key
```

Comment:
Was the api (on OpsGenie side) updated to support both?

Like 95% sure I tested this when writing it.